misc/misc.go

```diff
@@ -0,0 +1,26 @@
+package misc
+
+import (
+	"crypto/rand"
+	"math/big"
+)
+
+const seed = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz-"
+
+// RandString generates a random string of any length
+// and returns an error if an issue
+// was encountered while doing so.
```

Comment:
nit: this is given by the fact that the function returns an error, so I don't think you need to include this in the comment.

Explaining the functions error handling is important when it is non standard, i.e. errors are ignored or cause a panic.

Reply:
```suggestion
```